docs/cn/overview/Use-Cases.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: global
+title: 应用场景
+group: Overview
+priority: 6
+---
+
+* Table of Contents
+{:toc}
+
+世界上许多头部企业都在生产中部署Alluxio，以从数据中获取价值。我们在 [Powered-By](https://www.alluxio.io/powered-by-alluxio) 页面中列出了部分企业。下面我们将介绍一些最常见的 Alluxio 应用场景。
+
+## 应用场景1：加速云上分析和AI 作业
+
+<p align="center"> <img style="text-align: center" width="200" src="https://d39kqat1wpn1o5.cloudfront.net/app/uploads/2021/10/overview-case-1-cloud.png"/> </p>
+
+许多企业都在公有云（AWS S3、Google Cloud 或 Microsoft Azure）对象存储上运行分析和机器学习负载（Spark、Presto、Hive、Tensorflow 等）。
+
```

Comment:
remove extra empty line

Reply:
OK, I will fix it later.

Reply:
Probably because of the markdown syntax, if I delete the blank line, the preceding and following paragraphs become one paragraph. And when compiled by markdown, the extra blank lines are automatically deleted. So don't delete empty lines.